src/main/java/com/greenfox/poker/model/RegisterResponse.java

```diff
@@ -0,0 +1,47 @@
+package com.greenfox.poker.model;
+
+import org.springframework.stereotype.Component;
+
+@Component
+public class RegisterResponse {
```

Comment:
You'll use this response model in case of registration and login as well. So instead of naming this as `RegisterResponse`, something like `UserTokenResponse` would be more proper.